commands/container-registry/repository/repository.go

```diff
@@ -18,46 +31,52 @@ func RegRepoDeleteCmd() *core.Command {
 			Resource:  "repository",
 			Verb:      "repository",
 			Aliases:   []string{"rd", "del", "repo", "rep-del", "repository-delete"},
-			ShortDesc: "Delete all repository contents.",
-			LongDesc: "Delete all repository contents. The registry V2 API allows manifests and blobs to be deleted " +
+			ShortDesc: note + "Delete all repository contents.",
+			LongDesc: note + "Delete all repository contents. " +
+				"The registry V2 API allows manifests and blobs to be deleted " +
 				"individually but it is not possible to remove an entire repository. This operation is provided for " +
 				"convenience",
-			Example:    "ionosctl container-registry repository-delete --registry-id [REGISTRY-ID], --name [REPOSITORY-NAME]",
-			PreCmdRun:  PreCmdDelete,
-			CmdRun:     CmdDelete,
+			Example: "ionosctl container-registry repository-delete --registry-id [REGISTRY-ID], --name [REPOSITORY-NAME]",
+			PreCmdRun: func(c *core.PreCommandConfig) error {
+				fmt.Fprintf(c.Command.Command.ErrOrStderr(), note)
+				return PreCmdDelete(c)
+			},
+			CmdRun: func(c *core.CommandConfig) error {
+				fmt.Fprintf(c.Command.Command.ErrOrStderr(), note)
+				return CmdDelete(c)
+			},
 			InitClient: true,
 		},
 	)
 
-	cmd.AddStringFlag("name", "n", "", "Name of the repository to delete")
-	cmd.AddStringFlag("registry-id", "r", "", "Registry ID")
+	cmd.AddStringFlag(constants.FlagName, constants.FlagNameShort, "", "Name of the repository to delete")
+	cmd.AddStringFlag(constants.FlagRegistryId, constants.FlagRegistryIdShort, "", "Registry ID")
 	_ = cmd.Command.RegisterFlagCompletionFunc(
-		"registry-id", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		constants.FlagRegistryId,
+		func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 			return registry.RegsIds(), cobra.ShellCompDirectiveNoFileComp
 		},
 	)
-	return cmd
-}
 
-func CmdDelete(c *core.CommandConfig) error {
-	regId := viper.GetString("registry-id")
-	repoName := viper.GetString("name")
+	cmd.AddCommand(RepositoryDeleteCmd())
+	cmd.AddCommand(RepositoryListCmd())
+	cmd.AddCommand(RepositoryGetCmd())
 
-	res, _ := c.ContainerRegistryServices.Repository().Delete(regId, repoName)
-	if res.StatusCode == 204 {
-		fmt.Fprintf(c.Command.Command.OutOrStdout(), jsontabwriter.GenerateLogOutput("Repository is being deleted"))
-	} else {
-		fmt.Fprintf(c.Command.Command.OutOrStdout(), jsontabwriter.GenerateLogOutput("Repository could not be deleted"))
-	}
-
-	return nil
+	return cmd
 }
 
-func PreCmdDelete(c *core.PreCommandConfig) error {
-	err := core.CheckRequiredFlags(c.Command, c.NS, "name", "registry-id")
+func RepositoryNames(registryId string) []string {
+	repos, _, err := client.Must().RegistryClient.RepositoriesApi.RegistriesRepositoriesGet(
+		context.Background(),
+		registryId,
+	).Execute()
 	if err != nil {
-		return err
+		return nil
```

Comment:
is this change intended?

Reply:
yes, the `return err` is from a function that was originally in this file and that was moved to another file (repository/delete.go)